internal/core/services/workers_manager/metrics.go

```diff
@@ -0,0 +1,54 @@
+package workers_manager
+
+import (
+	"github.com/topfreegames/maestro/internal/core/monitoring"
+)
+
+var (
+	currentWorkersGaugeMetricOpts = monitoring.MetricOpts{
```

Comment:
nit: Is there any specific reason for defining it as a separated variable instead of inside the `CreateGaugeMetric` call?

Reply:
Copy&Paste reasons rsrs!